### ISSUE TITLE
Doesn't migrate

### ISSUE THREAD

**Issue by YoMe2** [zold-io/wts.zold.io#386]
Hi.  I have a wallet http://prntscr.com/wmvy02
It has balance.

Some time ago it was associated with my GitHub account - YoMe2.
And with my phone number 380 * * * * * * * * 5
I would log in and see my wallet and its balance.

Now I tried to login to WTS Zold with my phone number, whilst being signed in to my GitHub prior.
The system assigned me a new wallet.

I have a keygap for my original wallet, which is of 8 digits.
The keygaps examples stated in docs are longer. Maybe mine is shorter because it's older?

How can I get control of my original wallet? Thanks

**Comment by yegor256** [comment]
@YoMe2 yes, we had shorter keygaps earlier. Yours will work just fine too. Can you login using GitHub?
